fineract-provider/src/integrationTest/java/org/apache/fineract/integrationtests/AccountingScenarioIntegrationTest.java

```diff
@@ -786,18 +787,18 @@ public void checkPeriodicAccrualAccountingTillCurrentDateFlow() throws Interrupt
         final ArrayList<HashMap> loanSchedule = this.loanTransactionHelper.getLoanRepaymentSchedule(requestSpec, responseSpec, loanID);
         // MAKE 1
         List fromDateList = (List) loanSchedule.get(1).get("fromDate");
-        LocalDate fromDateLocal = LocalDate.now();
+        LocalDate fromDateLocal = LocalDate.now(ZoneId.systemDefault());
```

Comment:
It was just a warning from Error Prone that requires one to must  an explicit ZoneId( whether default). And I think it will be easier to replace ZoneId.systemDefault() with the tenant's timezone in another PR.